.azurepipelines/Platform-Build-Job.yml

```diff
@@ -58,6 +60,17 @@ jobs:
           Build.ArtifactsBinary: |
             **/QEMUQ35_*.fd
 
+        QemuQ35_NO_SMM_RELEASE:
+          Build.Package: QemuQ35Pkg
+          Build.File: "Platforms/QemuQ35Pkg/PlatformBuild.py"
+          Build.Arch: IA32,X64
+          Build.Flags: "BLD_*_SMM_ENABLED=FALSE"
+          Build.Target: "RELEASE"
+          Run: true
+          Run.Flags: $(run_flags_no_tests) BLD_*_QEMU_CORE_NUM=2 BLD_*_SMM_ENABLED=FALSE

```

Comment:
We don't have a real variable store (because of https://gitlab.com/qemu-project/qemu/-/issues/513) and this seems to break a number of the tests, so I think running them on regular Q35 is sufficient here.